### PR TITLE
Feature/mhip 241 birth date timezone issue

### PR DESCRIPTION
**Pull request by pradhanangan**

### CODE REVIEW

File: src/Domain/ValueObjects/BirthDate.cs [hunk 1]
```diff
@@ -101,5 +104,15 @@ protected override IEnumerable<object> GetEqualityComponents()
         {
             return birthDate.Value;
         }
+
+        private static DateTime DatetimeNowInAppTimeZone()
+        {
+            var now = DateTime.Now;
+            if (TZConvert.TryGetTimeZoneInfo(TIMEZONE, out var tzInfo))
```
[the remark this answers is not in the document]
Comment (pradhanangan) [reply src/Domain/ValueObjects/BirthDate.cs:111]: Hi @somithajosephHSL,

Used this library to support different platform (Windows Linux). Though, version of .NET 6.0 says it supports different platform
https://devblogs.microsoft.com/dotnet/cross-platform-time-zones-with-net-core/
, it did not work in Lambda when i deployed it (Lambda uses linux OS), so using this library.

[the remark this answers is not in the document]
Comment (somithajosephHSL) [reply src/Domain/ValueObjects/BirthDate.cs:111]: Using this in windows with IANA format seem to be working for me.

**Review by somithajosephHSL — APPROVED** [review]

File: src/Domain/ValueObjects/BirthDate.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (pradhanangan) [reply src/Domain/ValueObjects/BirthDate.cs]: I used Windows timezone in Lambda which did not work. Changed to IANA and remove the library.

[the remark this answers is not in the document]
Comment (somithajosephHSL) [reply src/Domain/ValueObjects/BirthDate.cs]: Perfect, Thanks

**Pull request merged**